src/helpers/query-daily-data.tsx

```diff
@@ -65,7 +65,7 @@ export async function queryPreviewDailyData(type: string, startDate: Date, endDa
 
 	// Modulo repeatable random numbers to get a value in range.
 	let currentDate = startDate;
-	while (currentDate <= endDate && currentDate < today) {
+	while (currentDate <= endDate && currentDate <= today) {
```

Comment:
This came up a few weeks ago, right?

Reply:
Yeah that's the one we discussed.

Reply:
Why wouldn't we want sample data from today?

Reply:
(I probably should have made a separate bugfix PR for that but it was such a small fix I just rolled it in here)

Reply:
Ah, sorry about that.  I totally read the diff backwards!  😬